alkemio.yml

```diff
@@ -77,6 +77,8 @@ security:
     # Cors allowed headers.
     allowed_headers: ${CORS_ALLOWED_HEADERS}:Origin,X-Requested-With,Content-Type,Accept,Authorization
 
+  encryption_key: ${ENCRYPTION_KEY}
```

Comment:
this `encryption_key` key is put at pretty much root level and is totally out-of-context to anything there, I'd suggest different name + category

Reply:
Agreed to be used as a general key for encryption, not only related to the VC.